### DISCUSSION TITLE
mocking subgraph fields

### DISCUSSION THREAD

**Discussion by tylercaceres** [Netflix/dgs-framework#1518] [Q&A]
Hello,
Is there any way to mock out the responses of a subgraph?
"message": "Validation error of type FieldUndefined: Field '<someField>' in type '<someType>' is undefined @ '<pathToField>'",

I tried using the mockprovider which works greats for fields defined in the schema.
However, is there any way to mock out fields that would be provided by a subgraph - this would be very helpful for localdevelopment. 🙏

**Comment by srinivasankavitha** [comment]
We don't have any functionality to support this in the framework. You would have to implement mock data fetchers in this case.
